alacritty/src/config/mod.rs

```diff
@@ -223,8 +223,11 @@ fn load_imports(config: &Value, config_paths: &mut Vec<PathBuf>, recursion_limit
         };
 
         if !path.exists() {
-            info!(target: LOG_TARGET_CONFIG, "Skipping importing config; not found:");
-            info!(target: LOG_TARGET_CONFIG, "  {:?}", path.display());
+            info!(
+                target: LOG_TARGET_CONFIG,
+                "Skipping importing config; not found:\n  {:?}",
```

Comment:
```suggestion
                "Config import not found:\n  {:?}",
```

`Skipping importing` just sounds strange. I think it's better to change this up a bit.

Though I personally still find it very strange that a single, self-contained import statement is logged on two lines. That just doesn't make a ton of sense to me.